docs_md/cardhovernet_tutorial.md

```diff
@@ -0,0 +1,121 @@
+# Running a SpatialOne Analysis using CARD & Hovernet
+
+For this tutorial, we assume the user is using the following path and has Docker properly set up: `/home/user/card_demo`. It is also assumed that the user has access to a machine with a GPU and the right memory set up.
+
+## 1. Retrieve the Code
+Donwload the code from the github repository
+```bash
+git clone https://github.com/Sanofi-Public/spatialone-pipeline.git
+cd spatialone-pipeline
+```
+
+## 2. Download Required Data
+
+### 2.1. Sample Data
+Download the two cancer samples, the reference data for deconvolution, and the default configuration files.
+```bash
+./download_experiment_data.sh
+```
+
+### 2.2. Downlaod Hovernet Weights
+Hovernet can work with different weights, as described in their [official repository]([https://drive.google.com/drive/folders/17IBOqdImvZ7Phe0ZdC5U1vwPFJFkttWp](https://github.com/vqdang/hover_net?tab=readme-ov-file#model-weights)). To use the CONSEP weights and labels described in the SpatialOne paper you should download them on the reference folder.
+
+You can manually download them from the [author's site](https://drive.google.com/drive/folders/17IBOqdImvZ7Phe0ZdC5U1vwPFJFkttWp), from Zenodo ([weights](https://zenodo.org/records/12801948/files/hovernet_original_consep_type_tf2pytorch.tar?download=1), [labels](https://zenodo.org/records/12801948/files/type_info.json?download=1)) or you can execute the following two instructions:
+```bash
+curl -L -o reference/hovernet_original_consep_type_tf2pytorch.tar "https://zenodo.org/records/12801948/files/hovernet_original_consep_type_tf2pytorch.tar?download=1"
+curl -L -o reference/type_info.json "https://zenodo.org/records/12801948/files/type_info.json?download=1"
+```
+
+
+## 3. Set Up the Environment
+Prepare your environment to build the container images if necessary. To that end, edit the `.env` file:
+```env
+HOST_DATA_PATH = /home/user/card_demo/spatialone-pipeline
+GPU_DEVICE_ID = 0
+```
+
+Export proxies only if necessary:
+```bash
+export HTTP_PROXY=http://your.proxy.url:port
+export HTTPS_PROXY=http://your.proxy.url:port
+```
+
+## 4. Build the Docker Image
+
+Build the docker containers:
+```bash
+make docker-build
+```
+Note that this operation may take up to 20 minutes
+
+## 5. Edit the Configuration Files
+
+Edit the config file located at `./conf/visium_config_flow.yaml` to reflect an appropriate configuration. You'll need to define the pipelines to run, and set up Hovernet & CARD.
```

Comment:
shouldn't this be the data folder and not the root one? {HOST_DATA_PATH}/conf/visium_config_flow

Reply:
If the user follows the tutorial step by step, they are both the same as HOST_DATA_PATH = /home/user/card_demo/spatialone-pipeline

Reply:
In any case, I have been explicit about it to avoid any confusion.